db/migrate/202002/location.yml

```diff
@@ -1,172 +1,286 @@
 # from https://github.com/ualbertalib/discovery/blob/master/config/locations.yml and https://www.neoslibraries.ca/member-libraries
```

Comment:
There have been changes to this since discovery got it's DB treatment.  I should probably go back and make sure the changes are intact.